docs/fsharp/language-reference/generics/index.md

```diff
@@ -91,6 +91,6 @@ There are two kinds of type parameters that can be used in F# programs. The firs
 - [Language Reference](../index.md)
 - [Types](../fsharp-types.md)
 - [Statically Resolved Type Parameters](statically-resolved-type-parameters.md)
-- [Generics in the .NET Framework](~/docs/standard/generics/index.md)
+- [Generics in the .NET Framework](../../../standard/generics/index.md)
```

Comment:
To match the name used in the C# sections
```suggestion
- [Generics](../../../standard/generics/index.md)
```

Reply:
I agree with @pkulikov on this change.

Once you make this, I'll :shipit:

Reply:
I unresolved this conversation. It doesn't appear that it's been made. (Not sure if that's due to the rebase or not)